setup.py

```diff
@@ -87,7 +87,7 @@
     license="MIT",
     zip_safe=False,
     keywords="ethereum blockchain evm",
-    packages=find_packages(exclude=["scripts", "tests", "tests.*"]),
+    packages=find_packages(exclude=["scripts.*", "tests", "tests.*"]),
```

Comment:
I still see it in the wheel after this change but I don't see it when we have both `scripts` and `scripts.*` in here. This might have given a false positive since we also removed the `__init__.py`.